pkg/infra/appcontext/user.go

```diff
@@ -50,13 +50,19 @@ func User(ctx context.Context) (*user.SignedInUser, error) {
 			case k8suser.APIServerUser:
 				fallthrough
 			case k8suser.SystemPrivilegedGroup:
+				orgId := int64(1)
 				return &user.SignedInUser{
 					UserID:         1,
-					OrgID:          1,
+					OrgID:          orgId,
 					Name:           k8sUserInfo.GetName(),
 					Login:          k8sUserInfo.GetName(),
 					OrgRole:        roletype.RoleAdmin,
 					IsGrafanaAdmin: true,
+					Permissions: map[int64]map[string][]string{
+						orgId: {
+							"*": {"*"}, // all resources, all scopes
```

Comment:
I am still getting:
```
kubectl create -f pkg/tests/apis/folders/testdata/folder-generate.yaml
Error from server: error when creating "pkg/tests/apis/folders/testdata/folder-generate.yaml": access denied to folder
```
but this fixed the list issues, and seems reasonable as a fallback for the super user

Reply:
I suspect that the problem is that folders does its own access checks and your signedinuser doesn't match those. Using the kubeconfig from the storage readme does work since you have a legit user